tools/serverpod_cli/lib/src/database/extensions.dart

```diff
@@ -358,7 +361,8 @@ extension IndexDefinitionPgSqlGeneration on IndexDefinition {
     var uniqueStr = isUnique ? ' UNIQUE' : '';
     var elementStrs = elements.map((e) => '"${e.definition}"');
 
-    out += 'CREATE$uniqueStr INDEX "$indexName" ON "$tableName" USING $type'
+    out +=
```

Comment:
Suggestion: Remove if not exists.

The same motivaiton here, hitting to broad when silencing migration failures.

Reply:
Same comment as above